pkg/metrics/eventcachemetrics/eventcachemetrics.go

```diff
@@ -33,13 +33,19 @@ var (
 		Help:        "The total of errors encountered while fetching process exec information from the cache.",
 		ConstLabels: nil,
 	}, []string{"error"})
+	eventCacheRetriesTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: consts.MetricsNamespace,
+		Name:      "event_cache_retries_total",
+		Help:      "The total number of retries for event caching per event type.",
+	}, []string{"event_type"})
```

Comment:
Could you give this label a different name? `event_type` typically corresponds to [EventType](https://github.com/cilium/tetragon/blob/f08c16f6fff107dba829b3d961c602f85158221b/api/v1/tetragon/events.proto#L18) from Tetragon API. Maybe `entry_type` would be more suitable here.